templates/database.yml.epp

```diff
@@ -30,6 +30,6 @@
   username: <%= $username %>
 <% } -%>
 <% if $password { -%>
-  password: "<%= $password %>"
+  password: <%= stdlib::to_ruby($password) %>
```

Comment:
On my phone so tricky to check, but can this be a `Sensitive` and does it handle that well?

Reply:
Technically, it can. No idea how to test this tho :see_no_evil: 

In pulpcore, we're lucky, it's a `String`.

Reply:
I think you can pass `Sensitive` to epp and it will automatically unwrap it, so that's fine.

Reply:
added a test, let's see

Reply:
going to drop the test (as it's failing now) and add sensitive support later

Reply:
https://github.com/theforeman/puppet-foreman/pull/1190 says this does *not* work for Sensitive

Reply:
and this not even works properly, what the hell

Reply:
oh I'm stupid, I was running a non-patched installer :see_no_evil: